src/components/pages/resources/books/books.jsx

```diff
@@ -0,0 +1,82 @@
+import { StaticImage } from 'gatsby-plugin-image';
+import React from 'react';
+
+import Button from 'components/shared/button/button';
+import Link from 'components/shared/link/link';
+
+const items = [
+  {
+    image: (
+      <StaticImage
+        className="w-full rounded"
+        imgClassName="w-full"
+        src="./images/what-is-ebpf.jpg"
+        alt="What is eBPF? Liz Rice, O'Reilly, 2022"
+        width={280}
+        height={364}
+        loading="lazy"
+      />
+    ),
+    linkUrl: 'https://isovalent.com/ebpf/#form',
+  },
+  {
+    image: (
+      <StaticImage
+        className="w-full rounded"
+        imgClassName="w-full"
+        src="./images/lern-epf.jpg"
+        alt="Learning eBPF O'Reilly book by Liz Rice"
+        width={280}
+        height={364}
+        loading="lazy"
+      />
+    ),
+    linkUrl: 'https://isovalent.com/learning-ebpf/#form',
+  },
+  {
+    image: (
+      <StaticImage
+        className="w-full rounded"
+        imgClassName="w-full"
+        src="./images/security.jpg"
+        alt="O’Reilly Report Security Observability with eBPF"
+        width={280}
+        height={364}
+        loading="lazy"
+      />
+    ),
+    linkUrl: 'https://isovalent.com/ebpf-security/#form',
+  },
+];
+
+const Books = () => (
+  <section className="hero safe-paddings mt-32 lg:mt-24 md:mt-16 sm:mt-14" id="books">
+    <div className="container flex flex-col items-center px-3">
```

Comment:
```suggestion
    <div className="container flex flex-col items-center">
```
it seems here it is superfluous and makes no sense, the container already has the required indents, try to avoid adding indents for the container even if they are required, better inside wrap in an additional div and specify the indents for it